base/libgit2/types.jl

```diff
@@ -449,6 +449,16 @@ struct FetchHead
     ismerge::Bool
 end
 
+function Base.show(io::IO, fh::FetchHead)
+    println(io, "FetchHead:")
+    println(io, "Name: $(fh.name)")
+    println(io, "URL: $(fh.url)")
+    print(io, "OID: ")
+    show(io, fh.oid)
+    println(io)
+    println(io, "Is merged: $(fh.ismerge)")
```

Comment:
I would probably have said "Merged:" instead of "Is merged:", though I don't feel too strongly